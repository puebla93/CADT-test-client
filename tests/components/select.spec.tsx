import { render } from '@testing-library/react'
import Select from '../../components/select'

describe('Select component tests', () => {
    test('Select matches snapshot', async () => {
        const {container} = render(
            <Select></Select>
        );
        expect(container).toMatchSnapshot();
    });
})
