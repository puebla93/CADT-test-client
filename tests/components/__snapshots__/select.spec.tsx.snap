// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select component tests Select matches snapshot 1`] = `
<div>
  <select
    class="sc-bcXHqe hwNYZh"
  />
</div>
`;
